include/realtime_tools/realtime_server_goal_handle.h

```diff
@@ -46,8 +46,8 @@ class RealtimeServerGoalHandle
   ACTION_DEFINITION(Action);
 
   typedef actionlib::ServerGoalHandle<Action> GoalHandle;
-  typedef boost::shared_ptr<Result> ResultPtr;
-  typedef boost::shared_ptr<Feedback> FeedbackPtr;
+  typedef std::shared_ptr<Result> ResultPtr;
```

Comment:
This may be a little bit of a slippery slope. I suggest you guys coordinate with @clalancette on this

https://github.com/ros-controls/ros_controllers/issues/352

Long story short: we shouldn't even be hosting these typedefs here as they should come from actionlib where boost is still used.

Reply:
If I read up on the story correctly, it looks like this can be fixed in `melodic`. https://github.com/ros-controls/ros_controllers/issues/352#issuecomment-496619360

Afterwards, I should be able to delete the typedefs from this PR

Reply:
I'd opt for keeping this one on a separate MR if you don't mind. This makes the package unreleasable until the above one is solved. I'd like to push out a release of all these changes as soon as we are through the reviews.

Reply:
Undid shared_ptr changes in 218a5f23bec267d25d5d38689fc07fe2dc344c6b

Reply:
Also, opened ros/actionlib#145